.github/workflows/beagle-bone-blue-binaries.yml

```diff
@@ -0,0 +1,101 @@
+name: beagle-bone-blue-binaries
+on:
+  push:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '30 5,17 * * *'
+jobs:
+  latest:
+    runs-on: ubuntu-20.04
```

Comment:
it should use 
container: ardupilot/ardupilot-dev-base:latest that already have everything installed.